autonomy/chain/service.py

```diff
@@ -686,8 +687,8 @@ def get_reuse_multisig_payload(  # pylint: disable=too-many-locals
             "data": HexBytes(
                 bytes.fromhex(
                     multisig_instance.encodeABI(
-                        fn_name="swapOwner",
-                        args=[new_owners[1], service_owner, new_owners[0]],
+                        fn_name="removeOwner",
+                        args=[new_owners[0], service_owner, 1],
```

Comment:
Fixes the issue jose faced on friday, instead of swapping the last owner, we fist add all owners and remove the service owner as the safe owner in the last tx